sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -729,6 +743,38 @@ object Radar {
         locationManager.startTracking(options)
     }
 
+    /**
+     * Starts tracking the user's location in the background using Radar dashboard settings.
+     *
+     * @see [](https://radar.io/documentation/sdk/android#background-tracking-for-geofencing)
+     */
+    @JvmStatic
+    fun startTracking() {
+        if (!initialized) {
+            return
+        }
+
+        apiClient.getConfig(object : RadarApiClient.RadarGetConfigApiCallback {
```

Comment:
call

GET /config
parse the tracking options
startTracking like normal, but with a listenToServer flag for getting further updates on POST /track

Reply:
add this as a comment above this block?

Reply:
yeah, we can comment it. I didn't see any convention of commenting like this in this codebase so was opting for consistency